rustls-platform-verifier/src/verification/apple.rs

```diff
@@ -43,21 +43,40 @@ fn system_time_to_cfdate(time: pki_types::UnixTime) -> Result<CFDate, TlsError>
 /// A TLS certificate verifier that utilizes the Apple platform certificate facilities.
 #[derive(Debug)]
 pub struct Verifier {
+    // Extra trust anchors to add to the verifier above and beyond those provided by the
+    // platform via rustls-native-certs.
```

Comment:
Sorry, one nit I missed: We don't use `rustls-native-certs` on Apple platforms. This should say "system-provided trust stores" or similar instead.

Reply:
Oh yeah I missed that too, I reword the comment